app/controllers/publishers_controller.rb

```diff
@@ -109,6 +114,35 @@ def update
     end
   end
 
+  def request_two_factor_authentication_removal
+    publisher = Publisher.find_by_email(params[:email])
+    if publisher
+      flash[:notice] = t("publishers.two_factor_authentication_removal.request_success")
+      MailerServices::TwoFactorAuthenticationRemovalRequestEmailer.new(publisher: publisher).perform
+    else
+      flash[:notice] = t("publishers.two_factor_authentication_removal.request_success")
```

Comment:
If the publisher doesn't exist we will say the same thing to the user saying that the request was success?